pkg/sethandler/controller.go

```diff
@@ -244,8 +244,9 @@ func (setHandler *SetHandler) applyCpusetToContainer(containerID string, cpuset
 	trimmedCid := strings.TrimPrefix(containerID, "docker://")
 	var pathToContainerCpusetFile string
 	filepath.Walk(setHandler.cpusetRoot, func(path string, f os.FileInfo, err error) error {
-		if strings.HasSuffix(path, trimmedCid) {
+		if strings.Contains(path, trimmedCid) {
```

Comment:
why was this needed?

Reply:
Because in current configuration cpuset files is under docker-<container_id>.scope folder. So directory name does not end with container_id.